cpp/harvest/marc_issn_lookup.cc

```diff
@@ -105,25 +109,37 @@ struct SubFieldInfo {
 };
 
 
-void PrettyPrintSubFieldInfo(const SubFieldInfo &sfi) {
-    std::cout << "t: " << sfi.t_ << std::endl;
-    std::cout << "w: " << sfi.w_ << std::endl;
-    std::cout << "x: " << sfi.x_ << std::endl;
-    std::cout << "online: " << (sfi.is_online_ ? "yes" : "no") << std::endl;
-    std::cout << "valid: " << (sfi.is_valid_ ? "yes" : "no") << std::endl;
+void PrettyPrintCacheEntry(const CacheEntry &ce) {
+    std::cout << "ppn: " << ce.ppn_ << std::endl;
+    std::cout << "title (t): " << ce.title_ << std::endl;
+    std::cout << "w: "
```

Comment:
i think we can remove w from debug output since it's basically the ppn which we output 2 lines above.